### PR TITLE
CI GitHub pipeline (hotfix) update for fetching repo name

### PR DESCRIPTION
**Pull request by TerrenceMcGuinness-NOAA**
# Description

Simple bug fix to GitHub CI pipeline script for Parallel Works.
Introduced a bug on its last update for setting the repo name in advance of running the GitHub CLI command for getting the repo owner and branch name of PRs.

# Type of change
- [x] Bug fix (fixes something broken)
- [ ] New feature (adds functionality)
- [ ] Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics

- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? 

# How has this been tested?
Gets tested when update to default repo gets checked in.

### CODE REVIEW

File: .github/workflows/pw_aws_ci.yaml [hunk 1]
```diff
@@ -72,9 +72,9 @@ jobs:
       id: get-branch
       run: |
         pr_number=${{ github.event.inputs.pr_number }}
+        repo={{ github.repository }}
         if [ "$pr_number" -eq "0" ]; then
           branch=${{ github.event.inputs.ref }}
-          repo=${{ github.repository }}
```
Comment (DavidHuber-NOAA) [inline .github/workflows/pw_aws_ci.yaml]: I'm a little confused by the syntax used here with `{{}}` and `$`.  Is this Jinja or is this specific syntax for GitHub workflows?

To retain consistency, I'm guessing that this should be

```suggestion
        repo=${{ github.repository }}
        if [ "$pr_number" -eq "0" ]; then
          branch=${{ github.event.inputs.ref }}
```

Comment (TerrenceMcGuinness-NOAA) [reply .github/workflows/pw_aws_ci.yaml]: Yes ${{ gitsutff }} is specific GitHub syntax, and yes line 75 was a typo

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
committing type suggestion

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good!

**Pull request merged**